CODEOWNERS

```diff
@@ -2,6 +2,11 @@
 # Unless a later match takes precedence, these users will be requested
 # for review whenever someone opens a pull request.
 *       @AzureAD/AppleIdentityTeam
+# @AzureAD/AppleCIAMTeam owns any files in the */native_auth
```

Comment:
nit: can you please update the name of this team? it should be MSAL-ObjC-CIAM

Reply:
Done.